src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PerformTouchAction.java

```
@@ -0,0 +1,360 @@
+		pointY = (int)(((pointY / 100f) * appNativeHeight) + taskbarNativeHeight);
+		if (mobileDriver.getContext() != "NATIVE_APP") {
+			mobileDriver.context("NATIVE_APP");
+		}
+		try {
+			mobileDriver.tap(1, pointX, pointY, duration);
+		} catch (Exception e) {}
+		try {
+			Thread.sleep(waitAfter);
+		} catch (Exception e) {}
+		if (mobileDriver.getContext() != "WEBVIEW_1") {
+			mobileDriver.context("WEBVIEW_1");
+		}
+	}
+	
+	/**
+	 * It scroll to element then tap on his center.
+	 * It scroll only vertically so if element need to be scrolled horizontally it won't work
+	 * @param locator Use By selector, Example By.cssSelector('img.loaded')
+	 * @param index If locator find more than one element use index to access element you want
+	 * @param duration In milliseconds, Reccomend 500
+	 * @param waitAfter In milliseconds
+	 */
+	public void tapOnWebElement(By locator, int index, int duration, int waitAfter) {
+		JavascriptExecutor js = (JavascriptExecutor) driver;
+		WebElement element;
+		if (index > 0) {
+			List<WebElement> elements = driver.findElements(locator);
+			element = elements.get(index);
+		} else {
+			element = driver.findElement(locator);
+		}
+		int elementStartPointX = element.getLocation().getX();
+		int elementHeight = element.getSize().getHeight();
+		int elementWidth = element.getSize().getHeight();
+		int offSetY = loadedPageHeight - element.getLocation().getY() - appWebviewHeight;
+		if (offSetY < 0) {
+			offSetY = -offSetY;
+		} else {
+			offSetY = 0;
+		}
+		js.executeScript("window.scrollTo(0, "+element.getLocation().getY()+")");
+		int pointX = (elementStartPointX + (elementWidth / 2)) * ratio;
+		int pointY = (taskbarWebviewHeight + (elementHeight / 2) + offSetY) * ratio; 
+		if (mobileDriver.getContext() != "NATIVE_APP") {
+			mobileDriver.context("NATIVE_APP");
+		}
+		try {
+			mobileDriver.tap(1, pointX, pointY, duration);
+		} catch (Exception e) {}
+		try {
+			Thread.sleep(waitAfter);
+		} catch (Exception e) {}
```

Please use a more specific Exception that the try block is catching.
Also, please either throw the exception or log the exception in the catch block.